lib/govuk_design_system_formbuilder/traits/localisation.rb

```diff
@@ -4,28 +4,39 @@ module Localisation
     private
 
       def localised_text(context)
-        key = localisation_key(context)
+        return unless @object_name.present? && @attribute_name.present?
 
-        # `I18n.exists?(nil)` returns true when `config.i18n.fallbacks` is
-        # enabled, so only proceed if the key is present too
-        return nil unless key.present? && I18n.exists?(key)
-
-        I18n.translate(key)
+        I18n.translate(localisation_key(context), default: nil) ||
+          I18n.translate(localisation_key(context, html: true), default: nil).try(:html_safe)
```

Comment:
I feel that this might be a bit much for a single line, perhaps breaking it up might make it easier to grasp?

```suggestion
        localise(context) || localise_html(context)
      end

      def localise(context)
        I18n.translate(localisation_key(context), default: nil)
      end

      def localise_html(context)
        I18n.translate(localisation_key(context, html: true), default: nil).try(:html_safe)
      end
```

Reply:
I agree. Will make the amendments and push!